package.json

```
@@ -28,7 +28,8 @@
     "storybook": "start-storybook -p 6006 --no-manager-cache -s public",
     "storybook:dev": "rm -rf ./node_modules/.cache && start-storybook -p 6006 --no-manager-cache -s public",
     "build-storybook": "build-storybook -s public",
-    "deploy-storybook": "storybook-to-ghpages"
+    "deploy-storybook": "storybook-to-ghpages",
+    "sync-to-figma": "anima-storybook sync"
```

quick reminder for adding 
`--design tokens`
```suggestion
    "sync-to-figma": "anima-storybook sync --design-tokens ./design-tokens.json"
```